templates/jenkins-slave-defaults.erb

```diff
@@ -60,6 +60,7 @@ JENKINS_USERNAME=<%= @quoted_ui_user %>
 JENKINS_PASSWORD=<%= @quoted_ui_pass %>
 
 OTHER_ARGS="<%= '-disableSslVerification' if @disable_ssl_verification -%>"
+DISABLE_UNIQUE="<%= '-disableClientsUniqueId' if @disable_ssl_verification-%>"
```

Comment:
I believe this is using the wrong variable in the ERB template. Instead of `disable_ssl_verification` it should be `disable_clients_unique_id`